R/verify_avaln.R

```diff
@@ -30,15 +30,15 @@ verify_avaln<-function(data,id_col="STUDYID",AVALC_col="AVALC",AVALN_col="AVALN"
   check_AVALN <-
     data %>%
     select(.data[[id_col]],.data[[param_col]],.data[[AVALC_col]],.data[[AVALN_col]]) %>% 
+    filter(is.na(.data[[AVALN_col]])|.data[[AVALN_col]]=="")%>%
```

Comment:
You have a check above that the AVALN_col must be numeric.  So, the `AVALN_col == ""` should never occur.  Would it work to remove that part?

Reply:
Yes, that's correct, this part should be removed. Since match_format() will convert all variables to required numeric/character format, AVALN_col == "" would never occur.